script/test

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# script/test: Run the test suite for the application.
+
+set -e
+
+script/utils/check-html
```

Comment:
In the `scripts-to-rule-them-all` repo we usually run `script/update` at the top of the script. For this one, I get:

```shell
❯ ./script/test
==> Removing any existing build...
==> Building site...
Could not find nokogiri-1.14.3-arm64-darwin, html-proofer-5.0.6, async-2.4.2, io-event-1.1.7 in locally installed gems
Run `bundle install` to install missing gems.
```

So, I need to run `./script/setup` manually before continuing.

Reply:
Interesting! We don't do that in either of the project repos I've worked on, but I'm happy to add it. In `scripts-to-rule-them-all`, it looks like `update` is just wrapping around `bootstrap`, but I guess it might be good to include and use `update` by convention?

Reply:
Added this in a final commit